packages/transactions/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "outDir": "./dist",
-  "rootDir": "./src"
+  "include": ["./src"],
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
`noEmit: true` is in the base config - does it need to be in all of the app configs too?